main.go

```diff
@@ -43,14 +43,20 @@ func main() {
 	}
 
 	rdr := csv.NewReader(f)
+	rdr.FieldsPerRecord = 2
 	records, err := rdr.ReadAll()
 	if err != nil {
-		log.Fatal(err)
+		log.Fatalf("could not read %s: %s", *urlFile, err)
 	}
 
 	var wg sync.WaitGroup
 	for _, r := range records {
-		host, desc := r[0], r[1]
+		var host, desc string
+		host := r[0]
+		desc := r[1]
+		if desc == "" {
+			desc = host
+		}
```

Comment:
This could be written more concisely by leaving the original line and adding just the `if`:

``` go
host, desc := r[0], r[1]
if desc == "" {
	desc = host
}
```